bucket/networkmanager.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2021.1.0",
+    "description": "Tool for managing networks and network troubleshooting",
+    "homepage": "https://borntoberoot.net/NETworkManager/",
+    "license": "GPL-3.0-or-later",
```

Comment:
Where is explicitly stated it is or later?

Reply:
:facepalm: I looked up correct string and coped wrong one. Of course it is v3 only.